<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teachers</class>
 <widget class="QWidget" name="teachers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 250);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дисциплина</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="name_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="discipline_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="position_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="telephone_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(176, 196, 222);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(8XXXXXXXXXX)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
